SocialCareCaseViewerApi.Tests/V1/Gateways/Database/GetCaseStatusesByPersonIdTests.cs

```diff
@@ -0,0 +1,35 @@
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SocialCareCaseViewerApi.V1.Gateways;
+using SocialCareCaseViewerApi.V1.Helpers;
+using Microsoft.EntityFrameworkCore;
+using SocialCareCaseViewerApi.Tests.V1.Helpers;
+using System;
+
+namespace SocialCareCaseViewerApi.Tests.V1.Gateways.Database
+{
+    [TestFixture]
+    public class GetCaseStatusesByPersonIdTests : DatabaseTests
+    {
+        private DatabaseGateway _databaseGateway;
+        private Mock<IProcessDataGateway> _mockProcessDataGateway = new Mock<IProcessDataGateway>();
+        private Mock<ISystemTime> _mockSystemTime;
+
+        [SetUp]
+        public void Setup()
+        {
+            _mockSystemTime = new Mock<ISystemTime>();
+            _databaseGateway = new DatabaseGateway(DatabaseContext, _mockProcessDataGateway.Object, _mockSystemTime.Object);
+            DatabaseContext.ChangeTracker.QueryTrackingBehavior = QueryTrackingBehavior.NoTracking;
+        }
+
+        [Test]
+        public void WhenNoMatchingIDReturnsNull()
+        {
+            var (person, _) = CaseStatusHelper.SavePersonWithCaseStatusToDatabase(DatabaseContext);
+            var response = _databaseGateway.GetCaseStatusesByPersonId(person.Id);
+            response.Should().BeEmpty();
+        }
```

Comment:
Should we have a test for when there is a matching Id?

Reply:
Test added!